soda_machine.rb

```diff
@@ -7,12 +7,22 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.length
   end
 
   def find_soda(soda_brand)
+    if @soda_machine[:sodas].include? soda_brand
```

Comment:
What is `@soda_machine`?  You seem to be using it like a hash, but as far as I can see, it's not defined anywhere.  What objects do you have access to here?